subprojects/core/src/main/java/org/gradle/api/internal/tasks/execution/SkipUpToDateTaskExecuter.java

```diff
@@ -46,26 +44,25 @@ public SkipUpToDateTaskExecuter(TaskExecuter executer) {
 
     public void execute(TaskInternal task, TaskStateInternal state, TaskExecutionContext context) {
         LOGGER.debug("Determining if {} is up-to-date", task);
-        Timer clock = Time.startTimer();
         TaskArtifactState taskArtifactState = context.getTaskArtifactState();
 
         List<String> messages = new ArrayList<String>(TaskUpToDateState.MAX_OUT_OF_DATE_MESSAGES);
         if (taskArtifactState.isUpToDate(messages)) {
-            LOGGER.info("Skipping {} as it is up-to-date (took {}).", task, clock.getElapsed());
+            LOGGER.info("Skipping {} as it is up-to-date.", task);
             state.setOutcome(TaskExecutionOutcome.UP_TO_DATE);
             context.setOriginExecutionMetadata(taskArtifactState.getExecutionHistory().getOriginExecutionMetadata());
             return;
         }
         context.setUpToDateMessages(ImmutableList.copyOf(messages));
-        logOutOfDateMessages(messages, task, clock.getElapsed());
+        logOutOfDateMessages(messages, task);
 
         executer.execute(task, state, context);
     }
 
-    private void logOutOfDateMessages(List<String> messages, TaskInternal task, String took) {
+    private void logOutOfDateMessages(List<String> messages, TaskInternal task) {
         if (LOGGER.isInfoEnabled()) {
             Formatter formatter = new Formatter();
-            formatter.format("Up-to-date check for %s took %s. It is not up-to-date because:", task, took);
+            formatter.format("Up-to-date check for %s completed. It is not up-to-date because:", task);
```

Comment:
I think we can say this simpler now. Like `Task '%s' is not up-to-date.`.

Reply:
fixed.